### PR TITLE
fix: respect style on svg node

### PR DESCRIPTION
**Pull request by Jackie1210**
Close https://github.com/vercel/satori/issues/397

Just make svg mode work
https://satori-playground-git-fork-jackie1210-feat-mask-type.vercel.sh/?share=zVXfb5swEP5XLFdTVykCzK8ElOxh3bRO6tSHrp00RWoJGOLWwQic0CzK_76zExKgWfc0aQ8I-7vz3XefffYGxyKhOMTjhK2mOUKVXHM62WzUGKE5ZdlchuicWNa788EOrFki5z0sYVXBozWgKacvDarGn1hJY8lEDrZY8OUib6wRZ1n-VdJFpUw0l7RsTE_LSrJ0fSkAzFX-rnkWxc9ZKZZ5cim4KMF-lqbpISusumW_aIgcuwX92NfiW5ZGt9tp_mFnHlerbDcCV8b5ZIpzkdMpbsAVo_VH8QK4hSxERvo7mncyTTagCATdYVqlLvSy4HkFMeZSFqFp1nVt1I4hysy0LcsyFYl9yD0vYLaIqudmckg0xZ30CLEEMh3yIKRW3eVMqmzLipa3RRTTm_yuatV03Gvt_n1dgGLnES_m0bkWp3HbVfIqp1ajDay7wKEGqCJmZcwpitWaAHxQrJz1aK_3jMOewrRscPOogan4HaeZ5jvZPC5L_v6MXDxuX6c6AuiYtQM2BNpgl0vbomiNhm3syA-SFpGct90TcP9G3MDwrBEi3tDw7Gs_MIhrI8-98tx7YntGMLzyfYMQx78HmzNy_GviBEYQBIj4rjFyvUviOobjODC3DZ-4iLg-hFR2yyC-B_N2ip8nq9EqFRHLpfXAWU6j8sEObPisiz_Wo1r2zVDkrVCtczq0u4bDWerhaosI8bqg2iLPPU1ybGbHcULTqk1_x-1LGSUM7o12yGyP_b05dFNN8UnhetyJIm8FPdRWqOsa_aK0N4F97OHaH_a1g7eqUteUFAV0rSj0tQf-9ZxJYN3ZvMZPpGlF9V0BDqcWKewGamdy37ndM2B2VfyX-vZP0yl9bSjjtb62ZQyDnvJa4e65aeS1fGM0_N8FHput8zxWzwKMAWQr-OMBFoV6TiscbrBuJxyO4EnDu57DoasmCZ0tMxymEa_oANOFeGLqgoeHXtZ6BnHUo_h5MaMJDmW5pNsBltEMPOaUc1GLkid4-xs
Need to wait for `@resvg/resvg-js` to support `mask-type`

### CODE REVIEW

File: test/__image_snapshots__/border-test-tsx-test-border-test-tsx-border-directional-should-support-directional-border-1-snap.png
Comment (Jackie1210) [inline test/__image_snapshots__/border-test-tsx-test-border-test-tsx-border-directional-should-support-directional-border-1-snap.png:1]: @yisibl after upgrade `@resvg/resvg-js` to `2.4.1`, the result of this test case changed, any pointer?

Comment (yisibl) [reply test/__image_snapshots__/border-test-tsx-test-border-test-tsx-border-directional-should-support-directional-border-1-snap.png:1]: [resvg-js 2.4.1 uses resvg 0.29.0,](https://github.com/yisibl/resvg-js/blob/5bcd474d7c908b518d9c1b30f1cffff86f565edd/Cargo.toml#L32) which does not yet support `mask-type` and needs to wait for me to upgrade to resvg 0.32.0, which I don't have time for yet.

**Comment by shuding** [comment]
We can ship this first 👍

File: test/__image_snapshots__/svg-test-tsx-test-svg-test-tsx-svg-should-respect-style-on-svg-node-1-snap.png
Comment (shuding) [inline test/__image_snapshots__/svg-test-tsx-test-svg-test-tsx-svg-should-respect-style-on-svg-node-1-snap.png:1]: Maybe update the test to be something else until we have mask-type support?

**Comment by Jackie1210** [comment]
@shuding I updated the test so that we can make `style` apply to svg node first

**Comment by shuding** [comment]
Thanks!

**Pull request merged**